test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```

Comment:
Ignore folder 'test' in file '.gitignore'. you don't need to follow it anymore

Reply:
done